src/libs/HttpUtils.js

```diff
@@ -107,13 +96,8 @@ function xhr(command, data, type = CONST.NETWORK.METHOD.POST, shouldUseSecure =
         formData.append(key, val);
     });
 
-    let apiRoot = shouldUseSecure ? CONFIG.EXPENSIFY.SECURE_EXPENSIFY_URL : CONFIG.EXPENSIFY.URL_API_ROOT;
-
-    if (shouldUseStagingServer(stagingServerToggleState)) {
-        apiRoot = shouldUseSecure ? CONFIG.EXPENSIFY.STAGING_SECURE_EXPENSIFY_URL : CONFIG.EXPENSIFY.STAGING_EXPENSIFY_URL;
-    }
-
-    return processHTTPRequest(`${apiRoot}api?command=${command}`, type, formData, data.canCancel);
+    const url = ApiUtils.getCommandUrl({shouldUseSecure, command});
+    return processHTTPRequest(url, type, formData, data.canCancel);
```

Comment:
Perhaps we want to stick doing `${apiRoot}api?command=${command}` here in HttpUtils.js instead of having `getCommandUrl` 

I saw another place that uses `{root}api?command=...` so it might be handy to have something reusable like `getCommandUrl`

Reply:
Either seem fine to me. Could go "rule of 3" on this one.